docs/pages/client_usage.mdx

```diff
@@ -0,0 +1,232 @@
+```
+
+**Note:** When you set a federation discovery URL within a `pelican://` URL, within configuration files, or with the `-f` flag, do not expect the client to work if the discovery URL contains a path (e.g. `osg-htc.org/PathToMyFederation`). The client will assume that the path provided is the namespace prefix for the object which will end in failure.
+
+**Note:** All object paths in a federation begin with a preceding `/`, and no relative paths are allowed.
+
+**Origins:**
+All objects in a federation live in some *origin*. Origins act like a flexible plug mechanism that exposes different type of storage backends to the federation. For example, the POSIX filesystem on most Linux computers is one type of storage backend an origin can expose to the federation. Other types of backends include S3 or HTTP servers, and Pelican plans to add many more. In most cases, a user does not need to know the particular backend used to store the object to download it from the federation.
+
+**Tokens and JWT:**
+Some namespace prefixes are public, like `/ospool/uc-shared/public`, while others are protected (i.e. they require authentication). Objects in public namespaces can be downloaded by anybody, but downloading objects from protected namespaces requires you prove to the origin supporting that namespace that you are allowed to access the object. In Pelican, this is done using signed JSON Web Tokens, or *JWT*s for short. In many cases, these tokens can be generated automatically.
+
+## The Different Pelican URL Schemes
+When running client commands with a request URL, there are three different URL schemes a user can use: `pelican`, `osdf`, and `stash`. This page uses examples with the `pelican://` URL scheme. Below is a description of the three URL schemes and what to expect when they are used:
+
+### The `pelican://` URL Scheme:
+When using the `pelican://` scheme, pelican does not know what federation to communicate with and therefore, the user must specify their federation URL either as the host-name in the URL itself or within the `-f` flag. The examples below show how you can format your request URLs in both of these ways.
+
+### The` osdf:///` URL Scheme:
+When using the `osdf:///` scheme, pelican will load in the federation metadata automatically with the Open Science Data Federation (OSDF)  defaults. This scheme should only be used if your federation or data can be accessed through the OSDF. When using this scheme, you should not specify the federation URL at all, as Pelican will handle this automatically. For example:
+```bash
+pelican object get osdf:///<namespace-prefix></path/to/file> <local/path/to/file>
+```
+This scheme has three slashes (`///`) after the `osdf` because the hostname is left empty to be automatically populated therefore, just start the URL with the namespace prefix. Pelican also supports the `osdf://` scheme (with two slashes) in case the user forgets to pass the third slash, but this is not recommended.
```

Comment:
Nope! That's why this sentence is there: `Pelican also supports the `osdf://` scheme (with two slashes) in case the user forgets to pass the third slash, but this is not recommended.`

We still allow users to make the mistake of having only two `//` after osdf: and everything should work fine